mavproxy/source/docs/getting_started/download_and_installation.rst

```diff
@@ -37,45 +40,41 @@ within Windows. It generally provides better stability than the native Windows i
 Linux
 =====
 
-MAVProxy can be installed via package managers. Select the appropriate set of commands for your Linux variant below.
+MAVProxy runs within Python 2 or 3. If your operating system version is less than 2 years old, it
+is recommended to use the Python 3 environment. Otherwise use Python 2.
 
-For Debian based systems (including WSL):
+For Python 3 on Debian based systems (including Ubuntu, WSL, Raspian):
 
 .. code:: bash
 
-    sudo apt-get install python3-dev python3-opencv python3-wxgtk4.0 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml
-    pip3 install mavproxy --user
+    sudo apt-get install python3-dev python3-opencv python3-wxgtk4.0 python3-pip python3-matplotlib python3-lxml python3-pygame
+    pip3 install PyYAML mavproxy --user
     echo "export PATH=$PATH:$HOME/.local/bin" >> ~/.bashrc
-    
-For Fedora based systems:
+
+For Python 2 on Debian based systems (including Ubuntu, WSL, Raspian):
 
 .. code:: bash
 
-    sudo dnf install python3-devel python3-opencv wxPython python3-pip python3-matplotlib pygame python3-lxml python3-yaml redhat-rpm-config
-    pip3 install mavproxy --user
+    sudo apt-get install python-dev python-opencv python-wxgtk4.0 python-pip python-matplotlib python-lxml python-pygame
+    pip3 install PyYAML mavproxy --user
     echo "export PATH=$PATH:$HOME/.local/bin" >> ~/.bashrc
-
-For Raspian / Raspberry Pi OS:
+    
+For Python 3 on Fedora based systems:
 
 .. code:: bash
 
-    sudo apt-get install python3-dev python3-opencv python3-wxgtk4.0 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml libxml2-dev
-    pip3 install mavproxy --user
+    sudo dnf install python3-devel python3-opencv python3-wxpython4 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml redhat-rpm-config
+    pip install PyYAML mavproxy --user

```

Comment:
same question here but re Python2....pip and not pip3?

Reply:
Correct. Copy-pasting to wrong section of text. Now fixed.